netbox/netbox/views/generic/bulk_views.py

```diff
@@ -615,6 +615,12 @@ def _update_objects(self, form, request):
             if form.cleaned_data.get('remove_tags', None):
                 obj.tags.remove(*form.cleaned_data['remove_tags'])
 
+            # Add/remove tagged VLANs
+            if form.cleaned_data.get('add_tagged_vlans', None):
+                obj.tagged_vlans.add(*form.cleaned_data['add_tagged_vlans'])
+            if form.cleaned_data.get('remove_tagged_vlans', None):
+                obj.tagged_vlans.remove(*form.cleaned_data['remove_tagged_vlans'])
+
```

Comment:
Model-specific fields can't go into a generic view. We'll need to come up with a pattern for handling these generically, probably in a manner similar to how we handle many-to-many assignments.

Reply:
Good point, how about this?